src/Bindings/LuaServerHandle.cpp

```diff
@@ -143,7 +143,7 @@ cTCPLink::cCallbacksPtr cLuaServerHandle::OnIncomingConnection(const AString & a
 	cCSLock Lock(m_CSConnections);
 	m_Connections.push_back(res);
 
-	return std::move(res);
```

Comment:
What's the warning generated here? This should be valid I would have thought.

Reply:
`warning: redundant move in return statement [-Wredundant-move]`

[Return value optimization](https://sigcpp.github.io/2020/06/08/return-value-optimization)

Reply:
Well that warning is a bit useless, it's not exactly like this does any harm, and the criteria for when variables don't get moved  isn't a super simple mnemonic... But this change is fine.